code/modules/power/lighting.dm

```diff
@@ -134,9 +120,15 @@
 	icon_state = "bulb-construct-stage1"
 	anchored = 1
 	stage = 1
-	fixture_type = "bulb"
+	fixture_type = /obj/machinery/light/small
 	sheets_refunded = 1
 
+/obj/machinery/light_construct/small/update_icon()
+	switch(stage)
+		if(1) icon_state = "bulb-construct-stage1"
+		if(2) icon_state = "bulb-construct-stage2"
+		if(3) icon_state = "bulb-empty"

```

Comment:
Please put the contents of an if/else/for check under the check itself, for readability.